### PR TITLE
added css and js linters

### PR DESCRIPTION
**Pull request by mce777**
Stylelint (for CSS) and ESLint (for JavaScript) were added, configured, and executed. I made corrections to the codebase based on feedback from the linters. Both linters now run without throwing any errors.

### CODE REVIEW

File: .stylelintrc.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "stylelint-config-standard",
+  "rules": {
+    "indentation": "tab",
+    "unit-whitelist": ["em", "s", "px"],
+    "color-hex-length": "long",
+    "property-no-unknown": [ true, {
+      "ignoreProperties": [
+        "composes"
```
Comment (MartinHelmut) [inline .stylelintrc.json]: Wrong indention (compare to `.eslintrc.json`) and missing last empty line ;)

File: lib/components/organisms/AppDemo/AppDemo.js [hunk 2]
```diff
@@ -16,31 +16,27 @@ function AppDemo(props) {
...
+					handleInputChange={() => {console.log('onInputChange/appDemo being logged here');}}
+					handleDropdownChange={() => {alert('why you change once?');}} />
 			</div>
 			<div className={style.input1}>
 				<InputDropDown id="id2"
 					inputName="Target Currency"
 					name="whatever2"
 					type="text"
-					options={ [{name:"item1", value:"value1"}, {name:"item2", value:"value2"}]}
+					options={ [{name:'item1', value:'value1'}, {name:'item2', value:'value2'}]}
```
Comment (MartinHelmut) [inline lib/components/organisms/AppDemo/AppDemo.js]: Please configure to use one whitespace between `:` and the new value, from`name:'item1'` to `name: 'item1'`

File: package.json [hunk 3]
```diff
@@ -1,24 +1,30 @@
...
+  "version": "1.2.0",
   "description": "currency styles",
   "main": "./styleguide/build/bundle.js",
   "dependencies": {},
   "devDependencies": {
     "babel-cli": "^6.18.0",
     "babel-preset-latest": "^6.16.0",
     "babel-preset-react": "^6.16.0",
+    "eslint": "^3.10.2",
+    "eslint-plugin-react": "^6.7.1",
```
Comment (MartinHelmut) [inline package.json]: Same here, please use same indention like in other `.json` files.

**Comment by mce777** [comment]
made the above mentioned corrections and updated pull request.

**Pull request merged**